src/store/reducers/inCartReducer.js

```diff
@@ -12,6 +12,16 @@ export default function (state = initialState, action) {
                 ...state,
                 cart: state.cart.concat(action.payload)
             }
+        case DELETE_CURRENT_PHONE_CART:
+            return {
+                ...state,
+                cart: action.payload
```

Comment:
supposing the payload is the itemIndex (as mentioned in a comment at CartsContainer.js)
```suggestion
                cart: state.cart.filter((_, index) => index !== action.payload)
```